pkg/pillar/types/assignableadapters.go

```diff
@@ -533,6 +533,9 @@ func (aa *AssignableAdapters) CheckBadUSBBundles() {
 			continue
 		}
 
+		if strings.HasPrefix(ioBundle.Error, "ioBundle collision:") {
```

Comment:
Can we use a string constant here? Or at least point to the place, where this error is set?
https://github.com/lf-edge/eve/blob/06869fa9ffa8892e1d3b35e4ae3f7ab671e29d9d/pkg/pillar/types/assignableadapters.go#L551

Reply:
So, do we clean the error always when we run the check?

Reply:
yes and we set it again, if the problem persists

Reply:
ok, get it

Reply:
> Can we use a string constant here? 

Done